### DISCUSSION TITLE
sync errors

### DISCUSSION THREAD

**Discussion by sbec101** [opengisch/QField#2850] [Q&A]
Hi, I have been successfully using qfield on android galaxy active tab 3 for a few weeks now and now I can't sync data, exporting a project is unreliable and getting python errors. Nothing from my end has changed, same old same old. I am using qfield 2.0.16 arctice fox, android 11 operating system. We are not using qfield cloud, just packaging to dropbox, ( old school for now)
![sync errors or lock](https://user-images.githubusercontent.com/102322524/166342524-009036d7-6d1e-486d-8e3c-a9f34fe21be1.PNG)

**Comment by suricactus** [comment]
@sbec101 sorry to hear that. However, in order to help you, please be more aspecific about the python errors you are experiencing. Providing a test project AND steps to reproduce drastically increase the chances to have your issue sorted out.
